examples/handlers/request_data/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "gotham_examples_handlers_request_data"
+version = "0.0.0"
+authors = ["Nicolas Pochet <npochet@gmail.com>"]
+description = "A basic application showing the request components available"
+license = "MIT/Apache-2.0"
+readme = "README.md"
+publish = false
+
```

Comment:
I'm sorry this comment is super pedantic but we're really trying to keep everything the same hence it being here.

Could this please change to:

```
name = "gotham_examples_handlers_request_data"
description = "A basic application showing the request components available"
version = "0.0.0"
authors = ["Nicolas Pochet <npochet@gmail.com>"]
publish = false
```